src/Client/StratumClient.cs

```diff
@@ -19,12 +20,23 @@ public class StratumClient : Client {
         public StratumClient (string _endpoint, int _port) : base (_endpoint, _port) { }
 
         public async Task<BlockchainTransactionGet> BlockchainTransactionGet (int id, string transactionId) {
-            var json = "{ \"id\": " + id + ", " +
-                "\"method\": \"blockchain.transaction.get\", " +
-                "\"params\": [\"" + transactionId + "\"] }";
+            var stringBuilder = new StringBuilder();
+            var stringWriter = new StringWriter(stringBuilder);
+            var jsonWriter = new JsonTextWriter(stringWriter);
+            jsonWriter.WriteStartObject();
+            jsonWriter.WritePropertyName("id");
+            jsonWriter.WriteValue(id);
+            jsonWriter.WritePropertyName("method");
+            jsonWriter.WriteValue("blockchain.transaction.get");
+            jsonWriter.WritePropertyName("params");
+            jsonWriter.WriteStartArray();
+            jsonWriter.WriteValue(transactionId);
+            jsonWriter.WriteEnd();
+            jsonWriter.WriteEndObject();
+            var json = stringBuilder.ToString();
```

Comment:
this is one way of using Newtonsoft.Json to write the JSON string, but maybe it's better that, in order to be consistent with how the deserialization is done, create a record type for the object to serialize to JSON? Example:

```
type Request =
    {
        Id: int;
        Method: string;
        Params: seq<obj>;
    }
```
(and adding attributes to map to their lowercase counterparts)

Reply:
(then later use `JsonConvert.SerializeObject()`)

Reply:
Ah, this is what I was asking about in [my comment](https://github.com/nblockchain/TcpEchoSharp/pull/3/files#diff-95bf908641a93f7247c4e74ec3b4cd4aR30) :)

Though it was dumb of me to ask about it when I had an example already sitting there in the code.